src/main/java/lotto/generator/LottoGenerator.java

```diff
@@ -1,7 +1,45 @@
 package lotto.generator;
 
 import lotto.domain.Lotto;
+import lotto.domain.LottoNumber;
+import lotto.view.OutputView;
 
-public interface LottoGenerator {
-    abstract Lotto create();
+import java.util.*;
+import java.util.stream.Collectors;
+import java.util.stream.IntStream;
+
+public class LottoGenerator {
+    private static List<LottoNumber> totalRandomNumbers;
+
+    static {
+        totalRandomNumbers = IntStream.rangeClosed(LottoNumber.MIN_LOTTO_NUMBER, LottoNumber.MAX_LOTTO_NUMBER)
+                .mapToObj(LottoNumber::new)
+                .collect(Collectors.toList());
+    }
+
+    public static Lotto create() {
+        Collections.shuffle(totalRandomNumbers);
+        Set<LottoNumber> randomNumbers = new TreeSet<>();
```

Comment:
subList를 활용하면 좀더 간결해질거에요
추가로 Lotto 생성자 인자로 List도 추가하면 클라이언트에서 new HashSet으로 감싸주지 않아도 되겠네요
```suggestion
        List<LottoNumber> lottoNumbers = new ArrayList<>(LottoNumber.lottoNumbers.values());
        Collections.shuffle(lottoNumbers);
        return lottoNumbers.subList(0, Lotto.LOTTO_NUMBERS_SIZE);
```